package.json

```diff
@@ -25,6 +25,7 @@
     "@types/react-dom": "^16.9.0",
```

Comment:
bitte deine PRs besser vorbereiten
1. Nur eine Sache pro PR. Hier machst du die Dropdowns und CSS refactoring
2. Kein umformatierungen. Wenn sich was ändert an den Formats mach das komplett in einem anderen PR. Der PR hier ist kaum lesbar

Reply:
Danke für das Feedback, werde ich in Zukunft beachten.